presto-orc/src/test/java/com/facebook/presto/orc/OrcTester.java

```diff
@@ -1680,7 +1680,6 @@ public static OrcSelectiveRecordReader createCustomOrcSelectiveRecordReader(
                 DwrfKeyProvider.of(intermediateEncryptionKeys));
 
         assertEquals(orcReader.getColumnNames().subList(0, types.size()), makeColumnNames(types.size()));
-        assertEquals(orcReader.getFooter().getRowsInRowGroup(), 10_000);
```

Comment:
why is this check removed?

Reply:
It blocks testing strides with different rows. I added a new test with stride length of 14,XXX rows to validate that row group conversion works as expected.